vllm/transformers_utils/tokenizer_group/__init__.py

```diff
@@ -23,7 +23,8 @@ def init_tokenizer_from_configs(model_config: ModelConfig,
                        max_input_length=None,
                        tokenizer_mode=model_config.tokenizer_mode,
                        trust_remote_code=model_config.trust_remote_code,
-                       revision=model_config.tokenizer_revision)
+                       revision=model_config.tokenizer_revision,
+                       do_lower_case=model_config.do_lower_case)
```

Comment:
Can we not add do_lower_case to the dict if it's not set? 

```suggestion
                       revision=model_config.tokenizer_revision)

    if model_config.do_lower_case is not None:
        init_kwargs["do_lower_case"] = model_config.do_lower_case
```

I think it's better if `"do_lower_case" in init_kwargs` evaluates to False if this parameter is not set